playbooks/provisioning/openstack/server-setup.yaml

```diff
@@ -0,0 +1,72 @@
+---
+- hosts: cluster_hosts
+  become: true
+  pre_tasks:
+  - include: pre_tasks.yml
+  roles:
+  - role: hostnames
+
+# Subscribe Hosts
+- hosts: cluster_hosts
+  become: true
+  roles:
+  - role: subscription-manager
```

Comment:
we would have to heavily enhance the rhel_subscribe role, as it doesn't deal with satellite.